python/ray/tests/test_advanced.py

```diff
@@ -591,7 +590,6 @@ def test_task_output_inline_bytes_limit(ray_start_cluster):
             "worker_lease_timeout_milliseconds": 0,
             "max_direct_call_object_size": 100 * 1024,
             "task_rpc_inlined_bytes_limit": 20,
-            "put_small_object_in_memory_store": True,
```

Comment:
I don't think this one was needed for this test, actually, since this is already dealing with task return inlining instead of put inlining, so this should be fine.